index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Messi Fever!</title>
+
+		<link rel="stylesheet" type="text/css" href="style.css">
+ 	</head>
+	
+	<body>
+		
+		<nav>
+			<a href="index.html"> Home </a> |
+			<a href="experience/experience.html" target="_blank"> Resume and Honors</a> |
+			<a href="http://www.fcbarcelona.com/all-about-leo-messi" target="_blank"> Official Messi site</a> |
+			<a href="http://www.worldsoccershop.com/shop-by-player-messi--lionel.html" target="_blank"> Buy Gear </a>
+		</nav>
+
+		<h1>Messi Fever! </h1>
+		<img src="lionel messi cover.jpeg" text="Lionel Messi" class="main" >
+		<h3>La Pulga "The Flea"</h3>
+		
+
+		<hr/>
+		<h3 class="flea">
+			What's that? The Flea? 
+			</h3>
+		
+ 		<p>
+			Yeah, Messi's nickname is "La Pulga," and that's Spanish for "the flea."
+
+			To us, it doesn't get much better than that. He's little. He harasses defenders. He never goes away and gets under your skin. It fits. <br/>
+
+			And anyway, if you can turn a wimpy-sounding nickname like "The Flea" into a descriptor of the 
+			<a href="https://www.genius.com/Lionel-messi-messis-career-honours-annotated" target="_blank"> best player in the world </a> , you've clearly done something right.
+			
+		</p>
+
+			<img class="action" src="in action messi.jpg" text="Lionel Messi" >
+		<p> 
+			Lionel Andres "Leo" Messi; born 24 June 1987) is an Argentine professional footballer who plays as a forward for Spanish club Barcelona and the Argentina national team. Often considered the best player in the world and rated by many in the sport as the greatest of all time, Messi is the only football player in history to win the FIFA Ballon d'Or five times, four of which he won consecutively, and the first player to win three European Golden Shoes. With Barcelona he has won seven La Liga titles and four UEFA Champions League titles, as well as three Copa del Rey titles. A prolific goalscorer, Messi holds the records for most goals scored in La Liga, a La Liga season, a calendar year, a single season, a Champions League match, and most Champions League seasons.
+			
+		</p>
+
+		<hr class="hr" />
+		Lithgow 2016
```

Comment:
:hammer: While this is technically valid html, this is text that is just "floating" around on the page. It can't be accessed properly with CSS. You can put this in a `<div>` to resolve the issue.